api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -409,7 +405,7 @@ RdrWorkspaceDemographic toRdrWorkspaceDemographics(
    * @param entity
    * @param idList
    */
-  private void updateDBRdrExport(RdrEntity entity, List<Long> idList) {
+  public void updateDBRdrExport(RdrEntity entity, List<Long> idList) {
```

Comment:
Is this worth a @VisibleForTesting annotation?

Reply:
completely forgot about this annotation. thank you!